examples/webgl_materials_nodes_playground.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+		<title>three.js webgl - node-editor playground</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link rel="stylesheet" href="fonts/open-sans/open-sans.css" type="text/css"/>
+		<link rel="stylesheet" href="fonts/tabler-icons/tabler-icons.min.css" type="text/css"/>
+		<link type="text/css" rel="stylesheet" href="main.css">
+  </head>
+  <body>
+	<script type="importmap">
```

Comment:
Should this example use the full specifier instead of an import map? It currently doesn't work in Firefox for this reason, while the other node material examples do work. I thought only the WebGPU examples were using import maps

Reply:
> Should this example use the full specifier instead of an import map?

@mattrossman It would be a considerable change, whole new node material system use imports map.

Reply:
@mrdoob @Mugen87 What do you think about this?

Reply:
Let's hope Firefox implements import maps soon 🤞

Reply:
Safari does not support import maps currently. I though the decision was, for now, to only support import maps on the webgpu examples?

https://caniuse.com/?search=importmap

Reply:
It was yes...
We can try to add relative paths at some point, but I think it's better to not worry about that for now.

Reply:
Import maps are now used in the webgl nodes examples, too. It's too soon, IMHO.

Reply:
Yeah... Lets reconsider next month.